src/components/AddToMetamask/index.tsx

```diff
@@ -1,10 +1,9 @@
-import React from 'react'
-import styled from 'styled-components'
 import { ChainId, Token } from '@kyberswap/ks-sdk-core'
-
 import MetaMaskLogo from 'assets/images/metamask.png'
 import { ButtonEmpty } from 'components/Button'
 import { RowFixed } from 'components/Row'
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
lol need to check this

Reply:
it sorted from nice order to alphabet

Reply:
should remove it

Reply:
we should have a common rule for sorting import 
And this rule is good

Reply:
Let's seek for other better rule